flocker/node/__init__.py

```diff
@@ -24,9 +23,8 @@
 
 __all__ = [
     'IDeployer', 'ILocalState', 'NodeLocalState', 'IStateChange',
-    'NoOp', 'NOOP_SLEEP_TIME',
```

Comment:
That was the number of seconds the flocker-container-agent would sleep between discovering local state.
The dataset agent has its own, longer, sleep settings.